joplin/templates/wagtailadmin/chooser/browse.html

```diff
@@ -0,0 +1,46 @@
+{% load i18n %}
+{% if page_types_restricted %}
+    {% if page_type_string == "base.departmentpage" %}
+    {% trans "Add a department that offers this page" as choose_str %}
+    {% include "wagtailadmin/shared/header.html" with title=choose_str  search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string icon="doc-empty-inverse" %}
+    {% elif page_type_string == "base.informationpage,base.servicepage" %}
+        {% trans "Select a page for the guide" as choose_str %}
+        {% include "wagtailadmin/shared/header.html" with title=choose_str  search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string icon="doc-empty-inverse" %}
+
+    {% else %}
+        {% trans "Choose" as choose_str %}
+        {% include "wagtailadmin/shared/header.html" with title=choose_str subtitle=page_type_names|join:", " search_url="wagtailadmin_choose_page_search" query_parameters="page_type="|add:page_type_string icon="doc-empty-inverse" %}
+
+    {% endif %}
+{% else %}
```

Comment:
just clarifying, this part means: if we don't have available page types set then use "choose a page" as our string.

Reply:
Yep!